Signal/src/call/UserInterface/CallUIAdapter.swift

```diff
@@ -63,6 +63,7 @@ extension CallUIAdaptee {
 
         guard self.callService.call == nil else {
             Logger.info("unexpectedly found an existing call when trying to start outgoing call: \(recipientId)")
+            //TODO terminate existing call.
```

Comment:
At some point in all my call testing, I found I was unable to place an outgoing call until the "current" call (which was invisible to the user) timed out ~2 minutes later.

So I'd hit the "call" button, and nothing would happen. I think if the user is attempting to place an outbound call we should end any current call so that the new one can go through.

@charlesmchen - if you think this is a good idea I can make a PR.